src/mod.ts

```diff
@@ -14,6 +12,8 @@ export const projectScripts = (args: string[]) => {
         `deno run -q --config=deno.jsonc --allow-read --allow-write --allow-net --allow-run  https://deno.land/x/${BUILDER_TAG}/mod.ts`,
       "start":
         `deno run -q --config=deno.jsonc --allow-read --allow-net ${startHookFlags} https://deno.land/x/${RUNTIME_TAG}/local-run.ts`,
+      "check-update":
+        `deno run -q --config=deno.jsonc --allow-read --allow-net https://deno.land/x/${HOOKS_TAG}/check-update.ts`,
```

Comment:
this one feels kinda meta if I'm thinking about it correctly - cause it's referencing itself - but I suppose it will always have the version set to it's latest version, is that the idea?

Reply:
Yeah, I called this same thought out and @filmaj added notes to the maintainer's guide so that as part of the release process we have to bump the version of hooks. You asking the question made me wonder if we can just remove the version entirely and always use the latest hooks version here?